templates/registry/markdown/weaver.yaml

```diff
@@ -9,6 +9,7 @@ acronyms:
   - AI
   - iOS
   - AWS
+  - Cicd
```

Comment:
make it sense to have here `CICD`? or `cicd`? Only first capital letter seems strange to me

Reply:
I'll adjust to be `CICD`